CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 ## Changelog
 
+### 0.15.0 [2021-04-04]
+
+#### General
+
+* Added CMake options to allow offline builds, i.e. without Googletest/ruby-rnp downloads.
+* Removed major library version from the library name (librnp-0.so/dll -> librnp.so/dll).
+* Improved handling of cleartext signatures, when empty line between headers and contents contains some spaces.
```

Comment:
"whitespace characters" would be more correct phrasing

Reply:
@antonsviridenko Thanks, fixed.